groups/instance/data.tf

```diff
@@ -70,7 +70,7 @@ data "aws_ami" "artifactory_ami" {
   most_recent = true
   owners      = [local.ami_owner_id]
   name_regex  = "${var.service}-${var.default_ami_version_pattern}"
-
+  
```

Comment:
This should just be an empty line

Reply:
Change implemented via: [
](https://github.com/companieshouse/artifactory-terraform/pull/20/commits/b658eab3c8890b731667e25b692d3da0e9851d06)